main.go

```diff
@@ -96,6 +97,10 @@ func (c *graphiteCollector) processLine(line string) {
 		name = labels["name"]
 		delete(labels, "name")
 	} else {
+		// if graphite.strict-match flag is set, we will drop this metric
```

Comment:
Comments should be full sentances, so:

If graphite.strict-match flag is set, we will drop this metric.